### ISSUE TITLE
[Request] Upgrade CapFrameX 

### ISSUE THREAD

**Issue by sanyer** [ScoopInstaller/Extras#10480]
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

capframex

### Description

 Frametime capture and analysis tool 

### Homepage

https://www.capframex.com/

### Download Link(s)

https://cxblobs.blob.core.windows.net/releases/CapFrameX_v1.7.1_Portable.zip

### Some Indication of Popularity/Repute

499 stars on GitHub

**Comment by sanyer** [comment]
I want to do some extra changes to this scoop - migrate it from using github to capframex.com
Should I create another PR for this issue?

**Comment by StarsbySea** [comment]
> I want to do some extra changes to this scoop - migrate it from using github to capframex.com Should I create another PR for this issue?

Yes, you can create a new issue stating a new problem and create a new pull request to close the previous issue. this issue can now be closed.

**Comment by aliesbelik** [comment]
Closing as per comments above.
